semcore/flex-box/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [5.19.0] - 2024-02-05
+
+### Added
+
+- `InvalidStatePattern` component.
```

Comment:
Let's name it `InvalidStateBox` to align it with `Box` naming.

Reply:
fixed